### ISSUE TITLE
Title: Incorrect Triton config generated for BGE reranker (wrong dtypes, wrong output name, invalid batching)

### ISSUE THREAD

**Issue by cop1cat** [cop1cat/triton-prep#3]
### Summary

When preparing the model `BAAI/bge-reranker-v2-m3` with `triton-prep`, the generated Triton `config.pbtxt` contains several incorrect fields:

1. Inputs are set to `TYPE_INT32`, but the ONNX model expects `INT64`.
2. Output tensor is named `"output"`, but the ONNX model exposes `"logits"`.
3. `preferred_batch_size` includes values larger than `max_batch_size`.
4. `max_queue_delay_microseconds` is set extremely low, which disables batching.

These issues lead to runtime errors in Triton Inference Server or to suboptimal performance.

---

### Steps to reproduce

```

triton-prep prepare BAAI/bge-reranker-v2-m3 
--output models/weights 
--model-name bge-reranker-v2-m3 
--device cpu 
--max-batch 8 
--dynamic 
--clean

```

Generated `config.pbtxt` contains:

```

input_ids: TYPE_INT32
attention_mask: TYPE_INT32
output name: "output"
preferred_batch_size: [4, 8, 16, 32]
max_batch_size: 8

```

---

### Expected Behavior

`triton-prep` should generate a config consistent with the ONNX model:

- `input_ids` and `attention_mask` must be `INT64`
- output name should match ONNX node name (`"logits"`)
- `preferred_batch_size` must not exceed `max_batch_size`
- batching delay should default to a reasonable value (e.g. 1000–2000 µs)

---

### Actual Behavior

- Triton refuses the model due to dtype mismatch  
- Output tensor `"output"` does not exist in the ONNX graph  
- Dynamic batching settings are ignored or invalid  
- Effective batching is disabled

---

### Environment

- triton-prep version: 0.1.0
- ONNX Runtime: 1.23+
- Triton Inference Server: 24.xx
- Python: 3.12
- Platform: Linux x86_64
